### DISCUSSION TITLE
Need help with the bisq monitor onboarding README

### DISCUSSION THREAD

**Discussion by embitkay** [bisq-network/bisq#5340] [Q&A]
Hi all, I'm currently working on fixing #5297 , but as a new developer to the bisq community, I'm finding it rather difficult to get my environment set up and haven't even got to being able to address the bug yet.

Is there a chance the author could walk me through the docs?

@freimair @ManfredKarrer @rex4539 ? Any of you want to help out...I see you all have had a role in this file.

https://github.com/bisq-network/bisq/blob/master/monitor/README.md

**Comment by ripcurlx** [comment]
Unfortunately neither @ManfredKarrer nor @freimair are active contributors anymore.

**Reply by embitkay** [reply]
Unfortunate indeed 😞. Well I just have to press forward. Looks @emzy came to the rescue though!

**Comment by Emzy** [comment]
I'm running the public monitor.
I was able to set it up using mainly this document: https://github.com/bisq-network/bisq/tree/master/monitor
I'm not sure you need all of that. 
I think the problem is somewhere at the point that the bisq-monitor (java) is sending the data to Graphite.

I can assist in setting it up if needed.

Alternatively I can try to get a clone of the productive system for you. If that would help.

**Reply by embitkay** [reply]
sorry @Emzy I did a terrible job at replying in time. Yeah, I had a lot of trouble getting setup but it may because I've barely used docker and have never used Grafite or Grafana before. If it isn't much trouble, then I'd prefer working in a cloned environment. Otherwise I'd also be fine with an assist. 

Let me know what works :)

**Reply by Emzy** [reply]
I will try to get a clone working. I think that will work best.

**Comment by freimair** [comment]
man that monitor does not look good at all...

I will take a look at it when I find some time, probably next week.

**Reply by Conza88** [reply]
Epic
